python/graphstorm/config/argument.py

```diff
@@ -3062,7 +3063,17 @@ def eval_metric(self):
                     if eval_metric.startswith(SUPPORTED_HIT_AT_METRICS):
                         assert eval_metric[len(SUPPORTED_HIT_AT_METRICS)+1:].isdigit(), \
                             "hit_at_k evaluation metric for classification " \
-                            f"must end with an integer, but get {eval_metric}"
+                            f"must end with an integer, but get {eval_metric}."
+                    elif eval_metric.startswith(SUPPORTED_RECALL_AT_PRECISION_METRICS):
+                        assert is_float(
```

Comment:
Better to check if the `beta` value is in (0, 1] here as the Configuration rst file requires.